docs/UserGuide.md

```diff
@@ -69,7 +71,8 @@ Scroll down to the bottom and click on `recruitIn.jar`.
 
     * **`delete`**`2` : Deletes the 2nd applicant shown in the list of all applicants, which would `Mary`.
 
-    * **`show`**`s/` : Displays all unique expected salaries in RecruitIn, which would be 1 result that is `3000`.
+    * **`show`**`s/` : Displays all unique expected salaries in RecruitIn, which would be 6 results that are 
```

Comment:
Would there not only be one unique salary shown since we added 2 applicants with s/3000 only?

Reply:
I think there have been some changes to the model data, it now returns 6 unique salaries when I ran it just now.